resources/runtime.rb

```diff
@@ -3,32 +3,38 @@
 property :get_pip_url, String, default: lazy { node['python3']['pip']['url'] }
 property :get_pip_checksum, String, default: lazy { node['python3']['pip']['checksum'] }
 property :pip_version, [String, TrueClass, FalseClass], desired_state: true, identity: true, default: lazy { node['python3']['pip']['version'] }
+property :pip_binary_name, [String, TrueClass, FalseClass], default: lazy { node['python3']['pip']['binary_name'] }
```

Comment:
Humm why `TrueClass` & `FalseClass` in addition to `String`? it's really weird to be able to pass a boolean or  a string, and now I see that it is used for most properties! 😨 
FAR: Why not `[String, NilClass]` ? it would be clearer & simpler; no need for all the ` || false`